amazon-cloudwatch/fluentd-config-map.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+data:
+  # Configuration is in Json format. No matter what configure change you make,
+  # please keep the Json blob valid.
+  cluster-info.json: |
```

Comment:
@martina-if, would you know where this format for `cluster-info` is coming from?
I was looking into:
```
Error: Couldn't find key logs.region in ConfigMap amazon-cloudwatch/cluster-info
```
that you were facing the other day, and after going over [the Amazon docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs.html#ContainerInsights-install-FluentD) again, I realised:
```console
$ kubectl create configmap cluster-info \
>     --from-literal=cluster.name=marcgitops \
>     --from-literal=logs.region=ap-northeast-1 -n amazon-cloudwatch
configmap/cluster-info created
```
generates something in a slightly different format:
```yaml
$ kubectl get cm cluster-info -n amazon-cloudwatch -o yaml
apiVersion: v1
data:
  cluster.name: marcgitops
  logs.region: ap-northeast-1
kind: ConfigMap
metadata:
  name: cluster-info
  namespace: amazon-cloudwatch
```

Reply:
Hm, it looks like one of the pods then adds `cluster-info.json` as it appears when reading the `ConfigMap` again a bit later. 🤔

Reply:
"Fixed" in https://github.com/weaveworks/eks-gitops-example/pull/10, but still interesting in more details to ensure this is the right "fix".